streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/SettingServiceImpl.java

```diff
@@ -61,27 +62,27 @@ public void initSetting() {
     @Override
     public boolean update(Setting setting) {
         try {
-            String value = setting.getSettingValue();
-            if (value != null) {
-                if (StringUtils.isEmpty(value.trim())) {
-                    value = null;
-                } else {
-                    value = setting.getSettingValue().trim();
-                }
-            }
+            String value = StringUtils.trimToNull(setting.getSettingValue());
             setting.setSettingValue(value);
             this.baseMapper.updateByKey(setting);
 
-            if (setting.getSettingKey().equals(CommonConfig.MAVEN_REMOTE_URL().key())) {
+            String settingKey = setting.getSettingKey();
+            if (CommonConfig.MAVEN_REMOTE_URL().key().equals(settingKey)) {
                 InternalConfigHolder.set(CommonConfig.MAVEN_REMOTE_URL(), value);
             }
-            if (setting.getSettingKey().equals(CommonConfig.MAVEN_AUTH_USER().key())) {
+            if (CommonConfig.MAVEN_AUTH_USER().key().equals(settingKey)) {
                 InternalConfigHolder.set(CommonConfig.MAVEN_AUTH_USER(), value);
             }
-            if (setting.getSettingKey().equals(CommonConfig.MAVEN_AUTH_PASSWORD().key())) {
+            if (CommonConfig.MAVEN_AUTH_PASSWORD().key().equals(settingKey)) {
                 InternalConfigHolder.set(CommonConfig.MAVEN_AUTH_PASSWORD(), value);
             }
-            settings.get(setting.getSettingKey()).setSettingValue(value);
+            if (settings == null || settings.size() == 0) {
+                initSetting();
```

Comment:
This logic is not as expected. H2, MySQL, and pgsql should be consistent. Data in data-h2.sql will be inserted into `t_ setting` table. These are completed in the system initialization phase, initSetting will only run once, So `initSetting` method should not be called again

Reply:
Thanks for your review, addressed.